exercises/circular-buffer/circular-buffer.js

```diff
@@ -0,0 +1,27 @@
+//
+// This is only a SKELETON file for the 'Circular Buffer' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+
+class CircularBuffer {
+  write() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  read() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  forceWrite() {
+    throw new Error("Remove this statement and implement this function");
+  }
+}
+
+export default () => {
+  // Complete this with the code for the exercise.
+  return new CircularBuffer();
+}
+
+export const BufferFullError = new Error("Error: Buffer is full");
+
+export const BufferEmptyError = new Error("Error: Buffer is empty");
```

Comment:
A stubbed error like this would be better as this does not give away the message implementation:

https://github.com/exercism/javascript/blob/d1fcd01c54c213b5a6116de0d9e296028e70b0f1/exercises/robot-simulator/robot-simulator.js#L6-L10

Reply:
Ah great, I hadn't found that example. Thanks!